### PR TITLE
BUG: REST API now spawned in conda env correctly

### PR DESCRIPTION
**Pull request by ebolyen**
@jakereps @Kleptobismol can you review?

### CODE REVIEW

**Comment by ebolyen** [comment]
Fixes #34

**Comment by jakereps** [comment]
👍  Job execution still working on OS X after this fix.

**Comment by ebolyen** [comment]
Awesome!

File: app/main.js [hunk 1]
```diff
@@ -9,9 +11,20 @@ let win;
...
-    const python = which.sync('python', { all: true }).find(path => path.search('conda') !== -1);
-    const api = spawn(python, ['-m', 'qiime_studio']);
+    // find the conda python and use that to infer the conda bin directory
+    // which needs to be reapplied to the front of our path so that subprocess
+    // spawns work correctly.
+    const python = which.sync('python', { all: true })
+                        .find(binaryPath => binaryPath.search('conda') !== -1);
+    const condaBin = path.dirname(python);
+
+    const api = spawn('python', ['-m', 'qiime_studio'], {
```
Comment (jakereps) [inline app/main.js]: Should this be a hard coded 'python' or using the python path from Line 17 like before?

Comment (ebolyen) [inline app/main.js]: Should be `python`, the full path was a horrible hack. If we can't launch the correct python then the PATH isn't getting set correctly.

**Pull request merged**